CHANGELOG

```diff
@@ -7,6 +7,13 @@ This file keeps track of all notable changes to the Slurm Charms.
 Unreleased
 ----------
 
+- fix breakage in charms when the slurmd leader is removed
+- remove slurmd peer relation in favor of the slurmd relation
+- slurmd units set their own inventory on the slurmd relation unit data 
+
+Unreleased
+----------
+
```

Comment:
```suggestion
- fix breakage in charms when the slurmd leader is removed
- remove slurmd peer relation in favor of the slurmd relation
```

We don't need to repeat the `Unreleased` header (and i guess it would break our release script).

Also I don't think it matters for end users where internal data lives. The CHANGELOG is copied to the osd-docs, to inform end users, not charm developers, of notable changes in the user-facing parts of project.

This should be written somewhere, i will start a "contributing" page with this info.